helpers/helpers.go

```diff
@@ -398,21 +398,27 @@ func CheckTokenExpiration(accessToken string) error {
 	}
 
 	switch untilExp := time.Until(expiration); {
-	case untilExp < 0:
-		return fmt.Errorf("the provided access token has expired, please renew it")
-	case untilExp > 0 && untilExp < 24*time.Hour:
-		fmt.Fprintln(
+	case untilExp >= 1*time.Minute && untilExp < 2*time.Hour:
+		fmt.Fprintf(
 			os.Stderr,
-			"The provided access token expires in",
-			time.Until(expiration).Truncate(time.Second),
+			"WARNING! The provided access token expires in only %d hour and %d minutes.\n",
```

Comment:
Maybe a bit over engineering, bit it's a bit weird to see 
```
The provided access token expires in only 0 hour and 29 minutes.
```
when the token is valid for <30 min

I would expect 
```
 The provided access token expires in only 29 minutes.
```

Reply:
It was quick to fix and I agree it looks better this way.